framework/filters/Cors.php

```diff
@@ -49,6 +49,8 @@
  *                 'Access-Control-Allow-Credentials' => true,
  *                 // Allow OPTIONS caching
  *                 'Access-Control-Max-Age' => 3600,
+ *                 // Allow the X-Pagination-Current-Page header to be exposed to the browser.
+ *                 'Access-Control-Expose-Headers' => ['X-Pagination-Current-Page'],
```

Comment:
@qiangxue just as an example of a comment. As you see [here](https://github.com/yiisoft/yii2/pull/6493/files#diff-7a9208d78ae9e05143782a4a99b017bfR87"), there is no default value in code.